site/app/templates/grading/electronic/PDFAnnotationBar.twig

```diff
@@ -56,6 +56,9 @@
 </div>
 
 <script>
+    $(document).ready(function () {
+        renderPDFToolbar();
```

Comment:
Is there a guarantee that this function will always run after loadPDFToolbar is run?

Reply:
The function should run before loadPDFToolbar. This is guarenteed because renderPDFToolbar is run on loading the Submission Panel, while loadPDFToolbar is run when a pdf file is expanded.